testfiles/_ResidentialBase.idf

```diff
@@ -112,6 +112,64 @@
     Yes,                     !- Use Weather File Rain Indicators
     Yes;                     !- Use Weather File Snow Indicators
 
+!-   ===========  ALL OBJECTS IN CLASS: SIZINGPERIOD:DESIGNDAY ===========
```

Comment:
If you try to run this file in the regression tool with Design Day Only enabled, it will fail because there aren't any design day objects present in the file.  Adding these won't affect the file's normal run at all, because the run control is not set to run design days.  But it does make it run happily for regressions :+1: